abstract_commands.xml

```diff
@@ -5,8 +5,26 @@
         is accessible, then \Fsize matching the register's actual size must be
         supported.
 
+        \begin{steps}{Perform the following sequence of operations:}
```

Comment:
Question about these bits in general... is Debug Module required to implement all these? Or is that an option that can be discovered (e.g. by writing all 1's and seeing what sticks?)